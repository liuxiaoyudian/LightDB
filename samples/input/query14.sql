SELECT Sailors.B, Sailors.C, Sailors.A FROM Sailors WHERE Sailors.B <= 200 AND Sailors.C = 50 AND Sailors.B >= 100;