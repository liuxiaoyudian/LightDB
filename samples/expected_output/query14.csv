200,50,1
100,50,4